Source/Extensions/Blazorise.DataGrid/_DataGridRow.razor.cs

```diff
@@ -173,6 +199,11 @@ protected internal Task OnCancelCommand()
         [Parameter] public EventCallback Cancel { get; set; }
 
         /// <summary>
+        /// Activates the multi select command.
+        /// </summary>
+        [Parameter] public EventCallback<(bool, TItem)> MultiSelect { get; set; }

```

Comment:
Instead of tupple maybe better would be to have a struct or class object?

Reply:
I like value tuples for simple things like this. 
I guess a struct is fine here, if you're looking for a bit more clarity. I'll make that change as requested.